src/tasks/get-items-by-category-task.ts

```diff
@@ -27,12 +28,18 @@ export class getItemsByCategoriesTask extends BaseCategoryTask<Item[]> {
     this.status = 'RUNNING';
 
     const { categoryIds } = this.input;
-    this.targetId = categoryIds?.join(',');
-    const items = await this.categoryService.getItemsByCategories(
-      categoryIds,
-      handler,
-    );
-
+    
+    const itemIdsList = await Promise.all(categoryIds.map(async (idString) => {
+      const categoryIdList = idString.split(',');
+      const itemIds = await this.categoryService.getItemsByCategories(
+        categoryIdList,
+        handler
+      ).catch((err) => []);
+      return itemIds;
+    })).catch((err) => [[]]);
```

Comment:
I don't think you should catch any error like this. I believe you did it just for the tests? But even the tests you are not checking anything at all. You should check at least that `this.categoryService.getItemsByCategories` run. If you can't make it we can debug it together.